api/v1/service.proto

```
@@ -0,0 +1,76 @@
+}
+
+message GetNodeByNameResponse {
+  Node node = 1;
+}
+
+service QueryService {
+  rpc Query(QueryRequest) returns (QueryResponse) {}
+}
+
+service CacheService {
+  rpc Cache(google.protobuf.Empty) returns (google.protobuf.Empty) {}
```

buf

"google.protobuf.Empty" is used as the request or response type for multiple RPCs.

(RPC_REQUEST_RESPONSE_UNIQUE)